src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -71,6 +78,7 @@ ManageCLIResponse.AccountCreated = Object.freeze({
 
 ManageCLIResponse.AccountDeleted = Object.freeze({
     code: 'AccountDeleted',
+    message: {}
```

Comment:
@naveenpaul1 @romayalon 
I want to verify that it is fine to add the property in the response (I don't know if you've got it structure from Scale and I want to know we have the flexibility to add it).